ambry-api/src/test/java/com.github.ambry/rest/MockBlobStorageService.java

```diff
@@ -108,6 +108,13 @@ public void handlePost(RestRequest restRequest, RestResponseChannel restResponse
     }
   }
 
+  /* Make sure PUT is not supported  */
+  @Override
+  public void handlePut(RestRequest restRequest, RestResponseChannel restResponseChannel) {
+    Exception exception = new RestServiceException("PUT is not supported", RestServiceErrorCode.UnsupportedHttpMethod);
```

Comment:
Mock does not do handlePrechecks or isUp. so cannot be similar to AdminiBlobStorageService.